client/state/signup/progress/schema.ts

```diff
@@ -29,6 +29,7 @@ export interface StepState {
 	providedDependencies?: string[];
 	status: 'completed' | 'processing' | 'pending' | 'in-progress' | 'invalid';
 	stepName: string;
+	wasSkipped?: boolean;
```

Comment:
The StepState object already uses the wasSkipped property for skipping the `plan` step https://github.com/Automattic/wp-calypso/blob/trunk/client/state/signup/progress/selectors.ts#L24